test/es-module/test-vm-source-text-module-leak.js

```diff
@@ -1,21 +1,24 @@
-// Flags: --experimental-vm-modules --max-old-space-size=16 --trace-gc
+// Flags: --expose-internals --experimental-vm-modules --max-old-space-size=16 --trace-gc
```

Comment:
It looks like this test is not using any internals, is it?
```suggestion
// Flags: --experimental-vm-modules --max-old-space-size=16 --trace-gc
```

Reply:
checkIfCollectableByCountingG() uses and internal utility.